test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -823,12 +823,49 @@ TEST_F(TransferAssetTest, InvalidWhenWrongPrecision) {
   ASSERT_FALSE(validateAndExecute());
 }
 
-TEST_F(TransferAssetTest, InvalidWhenDifferentCreator) {
```

Comment:
Why this test is deleted? Pr post doesn't describe this

Reply:
It was renamed